cps/main.c

```diff
@@ -1119,6 +1119,11 @@ run_cps(struct net_config *net_conf, struct gk_config *gk_conf,
 		cps_conf->log_ratelimit_interval_ms,
 		cps_conf->log_ratelimit_burst);
 
+	net_conf->gatekeeper_max_pkt_burst =
+		RTE_MAX(net_conf->gatekeeper_max_pkt_burst,
+			RTE_MAX(cps_conf->front_max_pkt_burst,
+				cps_conf->back_max_pkt_burst));
```

Comment:
We have to check that the back interface is enabled before accounting it in `gatekeeper_max_pkt_burst`.